ports/espressif/boards/heltec_esp32_wifikit_32_v2/mpconfigboard.mk

```diff
@@ -0,0 +1,47 @@
+# Product URL: disappeared/discontinued
+# Datasheet:    https://resource.heltec.cn/download/WiFi_Kit_32/WiFi%20Kit32.pdf
+#               There is apparently a typo with TX/RX (and other) GPIO descriptions
+#               in the Pin Description table: It should be GPIO1 / GPIO3 and all above 39 seem not to be available on esp32
+#
+# Schematic:    https://resource.heltec.cn/download/WiFi_Kit_32/WIFI_Kit_32_Schematic_diagram_V2.pdf
+#               https://resource.heltec.cn/download/WiFi_Kit_32/WIFI_Kit_32_Schematic_diagram_V2.1.PDF
+#               (same as V2, but V2.1 is better readable)
+# Pinout:   https://resource.heltec.cn/download/WiFi_Kit_32/WIFI_Kit_32_pinoutDiagram_V2.pdf
+#           https://resource.heltec.cn/download/WiFi_Kit_32/WIFI_Kit_32_pinoutDiagram_V2.1.pdf
+#           (exactly same as V2 [md5sum of pdf is 9ee18bb5abe0a41fe98825cc756e8e3e])
+
+CIRCUITPY_CREATOR_ID = 0x148E173C
+# TODO: What value should it be?
+CIRCUITPY_CREATION_ID = 0x00000000
```

Comment:
You can go here: https://github.com/creationid/creators/blob/main/creations/heltec.md and propose a PR with a new creation ID. It doesn't matter much what it is - if there's a stock number you could use that.

Reply:
Done.